config/config_test.go

```diff
@@ -673,19 +672,14 @@ func TestPrometheusValidateAndLog(t *testing.T) {
 			},
 		},
 		{
-			description: "[2] Port valid, Namespace invalid, Subsystem valid. Expect error",
+			description: "Port valid, Namespace invalid, Subsystem valid. Don't expect error",
```

Comment:
I don't think "invalid" is the right word when not set, that implies an error. I think `"Port valid, Namespace not set, Subsystem set. Don't expect error"` makes more sense

Reply:
Agree. Modified.